### DISCUSSION TITLE
5 point Screws_Tilt_Adjust

### DISCUSSION THREAD

**Discussion by Menethoran** [LeeOtts/Ender3v2-Klipper-Configs#21] [Ideas]
if i could suggest: A 5 point level rather than a 4 point level, the control being taken from bed center:

[screws_tilt_adjust]
screw1_name: Middle
screw1: 117.5,117.5
screw2: 57, 229
screw2_name: rear left screw
screw3: 227, 229
screw3_name: rear right screw
screw4: 227, 70
screw4_name: front right screw
screw5: 57, 70
screw5_name: front left screw
horizontal_move_z: 10
speed: 50
screw_thread: CW-M4

**Comment by LeeOtts** [comment]
I will look into it. Working on adding a linear rail to mine at the moment.  If I may ask, why do you suggest it?
